12_upgrade/05_caution.txt

```diff
@@ -33,6 +33,29 @@ configuration distributed to the nodes include the use of the new protocol.
 
 Rudder agents 4.1.x are compatible with 4.0 Rudder servers.
 
+[CAUTION]
```

Comment:
This is not related to the compatibility of 4.1 agents with older server, and should go in the upgrade notes instead.